src/components/DropDownMenu.svelte

```diff
@@ -1,37 +1,21 @@
 <script lang="ts">
   export let label: string;
   export let id: string = '';
-  export let options: Record<string, string>;
-  export let selected: string = '';
+  export let options: any[];
+  export let selected: any;
```

Comment:
nonblocker, but I have a feeling this type should not be any.

Reply:
Feel free to suggest a better type. It's a reusable component so it needs to work with an array. Each element can be formatted using the associated function.